src/Components/PageNotFound/PageNotFound.module.scss

```diff
@@ -7,7 +7,8 @@
 }
 
 .warning {
```

Comment:
Getting my feet into some CSS today was a good reminder of how much work goes into making this all look just right!